pallets/services-payment/src/lib.rs

```diff
@@ -214,8 +225,57 @@ pub trait OnChargeForBlockCredit<T: Config> {
     ) -> Result<(), Error<T>>;
 }
 
+pub struct ChargeForBlockCredit<Runtime>(PhantomData<Runtime>);
```

Comment:
Maybe we should add an extrinsic in this pallet that cleans the existing credits as well for a paraId, only callable by root, what do you think?